cachi2/core/models/output.py

```diff
@@ -45,6 +46,12 @@ def from_package_dict(cls, package: dict[str, Any]) -> "Component":
         )
 
 
+class SbomMetadata(pydantic.BaseModel):
+    """Metadata field in a SBOM."""
+
+    tools: list[dict[str, str]] = [{"vendor": "Red Hat", "name": "Cachi2"}]
```

Comment:
Can we have a Tool model?

(using the CycloneDX library is looking more and more tempting TBH)

Reply:
I started a sample commit with the library, but it proved to be more troublesome than I expected. File a story to analyse this later?